docs/core/diagnostics/metrics-collection.md

```diff
@@ -123,55 +127,56 @@ Press p to pause, r to resume, q to quit.
-cloud-native software. The built-in platform metric APIs are designed to be compatible with this
-standard to make integration straightforward for any .NET developers that wish to use it. At the time of writing, support for
-OpenTelemetry metrics is relatively new, but [Azure Monitor](/azure/azure-monitor/app/opentelemetry-overview)
-and many major APM vendors have endorsed it and have integration plans underway.
+[OpenTelemetry](https://opentelemetry.io/) is a vendor-neutral open-source project supported by the [Cloud Native Computing Foundation](https://www.cncf.io/) that seeks to standardize generating and collecting telemetry for cloud-native software. The .NET metric APIs are compatible with OpenTelemetry and make integration straightforward. [Azure Monitor](/azure/azure-monitor/app/opentelemetry-overview) and many major APM vendors have endorsed OpenTelemetry.
+
+This example shows one of the integrations available for OpenTelemetry metrics using the popular OSS [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) projects. The metrics data flow:
 
-This example shows one of the integrations available now for OpenTelemetry metrics using the popular OSS
-[Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) projects. The metrics data will flow like this:
+1. The .NET metric APIs collect measurements from the example app.
+1. The OpenTelemetry library running in the app aggregates the measurements.
+1. The Prometheus exporter library makes the aggregated data available via an HTTP metrics endpoint. 'Exporter' is what OpenTelemetry calls the libraries that transmit telemetry to vendor-specific backends.
+1. A Prometheus server:
 
-1. The .NET metric APIs collect measurements from our example application.
-2. The OpenTelemetry library running inside the same process aggregates these measurements.
-3. The Prometheus exporter library makes the aggregated data available via an HTTP
-metrics endpoint. 'Exporter' is what OpenTelemetry calls the libraries that transmit
-telemetry to vendor-specific backends.
-4. A Prometheus server, potentially running on a different machine, polls the
-metrics endpoint, reads the data, and stores it in a database for long-term persistence.
-Prometheus refers to this as 'scraping' an endpoint.
-5. The Grafana server, potentially running on a different machine, queries the data
-stored in Prometheus and displays it to engineers on a web-based monitoring dashboard.
+- Can run on a different machine
+- Polls the metrics endpoint
+- Reads the data
+- Stores the data in a database for long-term persistence. Prometheus refers to this as *scraping* an endpoint.
 
-### Configure the example application to use OpenTelemetry's Prometheus exporter
+1. The Grafana server, potentially running on a different machine, queries the data stored in Prometheus and displays it on a web-based monitoring dashboard.
```

Comment:
See [Overview](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-collection?branch=pr-en-us-35747#overview) and [original](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-collection?branch=pr-en-us-35747#overview)

@tdykstra  how can I have an unordered list in the middle of an ordered list? Adding two spaces in front of `-` doesn't help and gives a warning error.

Where can I put the code and how do I reference code snippets?

Reply:
Try indenting 3 instead of 2

Reply:
Here's the code location guidance: https://learn.microsoft.com/en-us/contribute/dotnet/dotnet-contribute#example-folder-structure